test/active_support_test_case_helper.rb

```diff
@@ -184,7 +184,12 @@ def disable_orchestration
 
   def read_json_fixture(file)
     json = File.expand_path(File.join('..', 'static_fixtures', file), __FILE__)
-    JSON.parse(File.read(json))
+    result = JSON.parse(File.read(json))
+    if file.start_with? "reports"
+      result["config_report"]
+    else
+      result
+    end
   end
```

Comment:
The patch reformats and corrects all our fixtures so they are actually valid payloads for the API call. Unfortunately, we use them in many tests where we actually build the call manually. Alternatively, I can change about 25 places in the codebase, but these tests will be deleted soon as well as this change.